packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -600,4 +600,40 @@
       <label>Ouvre dans un onglet du navigateur web</label>
       <name>quantic_OpensInBrowserTab</name>
     </customLabels>
+    <customLabels>
+        <label>Erreur du composant {{0}}</label>
+        <name>quantic_ComponentError</name>
+    </customLabels>
+    <customLabels>
+        <label>Consultez la console développeur pour plus d'informations.</label>
+        <name>quantic_LookAtDeveloperConsole</name>
+    </customLabels>
+    <customLabels>
+        <label>"{{0}}" est un nombre maximum de suggestions de documents non valide. Un nombre entier supérieur à 0 était attendu.</label>
```

Comment:
@mmitiche If you decide to go with my EN suggestion, here's the FR equivalent:

"Le nombre maximum de suggestions de documents doit être supérieur à 0."